pkg/apis/kiegroup/v1/types.go

```diff
@@ -31,12 +32,16 @@ type AppSpec struct {
 }
 
 type Environment struct {
-	Console OpenShiftObject   `json:"console"`
-	Servers []OpenShiftObject `json:"servers"`
+	Console CustomObject   `json:"console,omitempty"`
+	Servers []CustomObject `json:"servers,omitempty"`
 }
 
-type OpenShiftObject struct {
-	DeploymentConfig apiv1.DeploymentConfig `json:"deployment"`
-	Service          corev1.Service         `json:"service"`
-	Route            routev1.Route          `json:"route"`
+type CustomObject struct {
+	PersistentVolumeClaims []corev1.PersistentVolumeClaim `json:"persistentvolumeclaims,omitempty"`
+	ServiceAccounts        []corev1.ServiceAccount        `json:"serviceaccounts,omitempty"`
+	Secrets                []corev1.Secret                `json:"secrets,omitempty"`
+	RoleBindings           []authv1.RoleBinding           `json:"rolebindings,omitempty"`
+	DeploymentConfigs      []appsv1.DeploymentConfig      `json:"deploymentconfigs,omitempty"`
```

Comment:
When would a single object have multiple DCs?

Reply:
w/ a prod deployment, the kie servers also require postgres... which has its own dc.